### PR TITLE
Adding network in public transport steps

### PR DESCRIPTION
**Pull request by Guitouille**

### CODE REVIEW

File: navitiasdkux/src/main/java/org/kisio/navitia/sdk/ui/journey/common/JourneysRequest.java [hunk 1]
```diff
@@ -82,6 +82,7 @@ private JourneysRequest(Parcel in) {
         } else {
             addPoiInfos = null;
         }
+
```
Comment (EvenModiguy) [inline navitiasdkux/src/main/java/org/kisio/navitia/sdk/ui/journey/common/JourneysRequest.java]: Please remove this empty line.

File: navitiasdkux/src/main/java/org/kisio/navitia/sdk/ui/journey/roadmap/viewholder/PublicTransportStepViewHolder.java [hunk 2]
```diff
@@ -108,6 +113,14 @@ public void fillView(Context context, final RoadmapModel roadmapModel) {
         gIcon.setText(Icon.modeIcon(ModeUtil.getMode((section))));
         gTake.setText(Format.takeTheMode(context, section.getDisplayInformations().getPhysicalMode()));
 
+        String network = getNetwork(roadmapModel.getSection());
+        if(Configuration.MULTI_NETWORK) {
```
Comment (EvenModiguy) [inline navitiasdkux/src/main/java/org/kisio/navitia/sdk/ui/journey/roadmap/viewholder/PublicTransportStepViewHolder.java]: Please add a space right just after `if`.

File: navitiasdkux/src/main/java/org/kisio/navitia/sdk/ui/journey/common/JourneysRequest.java (hunk 1 above)
Comment (Guitouille) [reply navitiasdkux/src/main/java/org/kisio/navitia/sdk/ui/journey/common/JourneysRequest.java]: OK

File: navitiasdkux/src/main/java/org/kisio/navitia/sdk/ui/journey/roadmap/viewholder/PublicTransportStepViewHolder.java (hunk 2 above)
Comment (Guitouille) [reply navitiasdkux/src/main/java/org/kisio/navitia/sdk/ui/journey/roadmap/viewholder/PublicTransportStepViewHolder.java]: OK

**Pull request merged**